static/js/about_zulip.js

```diff
@@ -28,4 +30,25 @@ export function initialize() {
             page_params.zulip_merge_base !== page_params.zulip_version,
     });
     $(".app").append(rendered_about_zulip);
+
+    $("#about-zulip i.tippy-zulip-tooltip").on("click", (event) => {
+        show_tooltip($(event.target));
+    });
+}
+
+export function show_tooltip(element) {
+    // Display a tooltip to notify the user the version was copied.
+    const instance = tippy("#about-zulip i.tippy-zulip-tooltip", {
+        onUntrigger() {
+            remove_instance();
+        },
+    }).find((tippy) => tippy.id === element[0]._tippy.id);
+    instance.setContent($t({defaultMessage: "Copied!"}));
+    instance.show();
+    function remove_instance() {
+        if (!instance.state.isDestroyed) {
+            instance.destroy();
+        }
+    }
+    setTimeout(remove_instance, 3000);
```

Comment:
There is a file in `static/js/tippyjs.js` that handles specific tooltips that require custom JavaScript code or configuration. I wonder if we could move this function you've created into that file to keep everything clean.